ord_schema/validations.py

```diff
@@ -239,6 +315,32 @@ def validate_reaction_observation(message):
 
 def validate_reaction_workup(message):
     ensure_details_specified_if_type_custom(message)
+    if (message.type == reaction_pb2.ReactionWorkup.WAIT and
```

Comment:
Note that if you want to save space, `message.WAIT` should work as well.

Reply:
I thought about that, but wanted to use the full ```reaction_pb2```-derived variable for clarity since ```message``` isn't terribly informative.

Reply:
SGTM